sqlmesh/dbt/profile.py

```diff
@@ -43,13 +43,15 @@ def load(cls, context: DbtContext) -> Profile:
         Returns:
             The Profile for the specified project
         """
-        if not context.project_name:
+        if not context.profile_name:
             project_file = Path(context.project_root, PROJECT_FILENAME)
             if not project_file.exists():
                 raise ConfigError(f"Could not find {PROJECT_FILENAME} in {context.project_root}")
 
-            context.project_name = context.render(load_yaml(project_file).get("name", ""))
-            if not context.project_name:
+            context.profile_name = context.render(
```

Comment:
why are you loading the project file twice here? seems expensive

Reply:
true. Will fix.